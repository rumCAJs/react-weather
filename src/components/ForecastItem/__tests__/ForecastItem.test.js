import React from 'react'
import { ForecastItem } from '../index'
import renderer from 'react-test-renderer'

const data = [
  {
    dt_txt: '2019-09-25 09:00:00',
    main: {
      temp: 17
    },
    weather: [
      {
        icon: '04d'
      }
    ]
  }
]

it('renders correctly', () => {
  const tree = renderer.create(<ForecastItem data={data} />).toJSON()
  expect(tree).toMatchSnapshot()
})
