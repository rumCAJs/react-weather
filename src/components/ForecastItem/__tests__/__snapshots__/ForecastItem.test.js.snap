// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="styled__Wrapper-e48zi3-0 bkioAu"
>
  <div
    className="styled__Heading-e48zi3-2 eKSBRZ"
  >
    Wednesday
  </div>
  <div
    className="styled__DateRow-e48zi3-3 jCVlji"
  >
    25.9.2019
  </div>
  <div
    className="styled__Line-e48zi3-1 eNutno"
  >
    <span>
      09:00
    </span>
    <span
      className="styled__StyledTemp-zfc47z-0 VoffX"
    >
      17
      Â°C
    </span>
    <img
      className="WeatherIcon__StyledImg-qloq4m-0 kXqJBM"
      height={50}
      src="https://openweathermap.org/img/wn/04d@2x.png"
      width={50}
    />
  </div>
</div>
`;
